### ISSUE TITLE
Package Request: OpenRCT2

### ISSUE THREAD

**Issue by espectalll** [void-linux/void-packages#1014]
Every Linux distribution seeking to be worthy of such title needs to have OpenRCT2. This is the ultimate package that will unleash all of Void Linux's true potential and put it up there amongst the greatest distros. Void already has OpenTTD available, so this is, out of a doubt, the final milestone before achieving true computing nirvana.

I plan to package it in a moment, no warrants tho.

**Comment by espectalll** [comment]
OK, I won't have it ready for now. There's apparently an optional dependency on `discord-rpc` that may have to be packaged on its own - or maybe there's not that much demand for Discord integration in the first place.

**Comment by AluisioASG** [comment]
I got a [package half-done](https://github.com/AluisioASG/void-packages/blob/openrct2/srcpkgs/OpenRCT2/template) already  in case you want something to base from.

**Comment by lemmi** [comment]
I also made a template a while ago, but OpenRCT2 has 2-stage build process which made it confusing to cross compile.

**Comment by lemmi** [comment]
I also made a template a while ago, but OpenRCT2 has 2-stage build process which made it confusing to cross compile.

**Comment by Johnnynator** [comment]
Cross Compiling seems to be an easy fix.
```
diff --git a/srcpkgs/OpenRCT2/template b/srcpkgs/OpenRCT2/template
index cf9488ebc6..8ccc469d20 100644
--- a/srcpkgs/OpenRCT2/template
+++ b/srcpkgs/OpenRCT2/template
@@ -1,6 +1,6 @@
 # Template file for 'OpenRCT2'
 pkgname=OpenRCT2
-version=0.1.2
+version=0.2.1
 revision=1
 #only_for_archs="i686 x86_64"
 build_style=cmake
@@ -15,12 +15,23 @@ maintainer="Aluísio Augusto Silva Gonçalves <aluisio@aasg.name>"
 license="GPL-3.0-or-later"
 homepage="https://openrct2.io"
 distfiles="https://github.com/OpenRCT2/OpenRCT2/archive/v${version}.tar.gz"
-checksum=@808b3ce6eef1b3318dcf761f3c5a2e02e5e18361ae55b299f78f99dc50b545b6
+checksum=@f4099eb6c2f6027f8d24582707953643d0485f69c6c877afcd394149ea550557
 
 build_options="multiplayer"
 build_options_default="multiplayer"
 desc_option_multiplayer="Enable multiplayer support"
 
+if [ "$CROSS_BUILD" ]; then
+	hostmakedepends+=" OpenRCT2"
+	CXXFLAGS="-DHAVE_IMMINTRIN_H=false"
+fi
+
+pre_configure() {
+	if [ "$CROSS_BUILD" ]; then
+		sed -i -e 's#COMMAND ./openrct2#COMMAND openrct2#g' CMakeLists.txt
+	fi
+}
+
 #post_install() {
 #	#vmkdir usr/share/applications
 #	#vcopy "distribution/linux/*.desktop" usr/share/applications
```

**Comment by maxice8** [comment]
ping

**Comment by Johnnynator** [comment]
resolved with c722877ef8556a08ea68b681fea1e86ce982a94e
